lib/icinga/comment.cpp

```diff
@@ -65,7 +65,7 @@ void Comment::OnAllConfigLoaded()
 
 	Host::Ptr host = Host::GetByName(GetHostName());
 
-	if (GetServiceName().IsEmpty())
+	if (GetServiceName().IsEmpty() || ! host)
 		m_Checkable = host;
 	else
 		m_Checkable = host->GetServiceByShortName(GetServiceName());
```

Comment:
* Colleagues, wanna get this little thing done, so that OP can test #9752 with snapshot packages?

Reply:
https://github.com/Icinga/icinga2/pull/9861#issuecomment-2473668998:

> I maintain my own build because of this.

Sounds like it should already be tested?